content/attributes/group_permissions.yml

```diff
@@ -0,0 +1,13 @@
+---
+type: object
+
+description:
+  The permissions that the authenticated user has for a group.
+
+properties:
+  canInviteAsCollaborator:
+    type: boolean
+    description:
+      Specifies if the user can invite the group to collaborate on
+      folders (Create Collaboration).
```

Comment:
```suggestion
      any items.
```

Reply:
I assume this applies to files, folders, and anything that a group can really be collaborated on?

Reply:
Yep! I took the wording from invitability_level, so I'll change it there too